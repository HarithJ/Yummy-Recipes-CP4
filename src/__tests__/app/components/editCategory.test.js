import React from 'react';
import { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson, { shallowToJson } from 'enzyme-to-json';

import CategoryEditForm from '../../../components/categories/editCategory.js'

describe("category module", () => {
    const categoryEditForm = shallow(<CategoryEditForm />)
    it("renders properly", () => {
      expect(shallowToJson(categoryEditForm)).toMatchSnapshot();
    })

    it("has a function to handle editing of a category", () => {
      expect(categoryEditForm.instance().handleEditing(global.event));
    })

})
