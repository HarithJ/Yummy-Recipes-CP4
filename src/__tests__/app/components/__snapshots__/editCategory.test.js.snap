// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`category module renders properly 1`] = `
<form
  onSubmit={[Function]}
>
  <div
    className="row justify-content-center"
  >
    <div
      className="col-4"
    >
      <input
        className="form-control validate"
        onChange={[Function]}
        type="text"
      />
    </div>
    <div
      className="col-auto"
    >
      <button
        className="btn btn-warning"
        type="submit"
      >
        Edit Category
      </button>
    </div>
    <div
      className="col-auto"
    >
      <button
        className="btn btn-primary"
        onClick={[Function]}
      >
        Cancel
      </button>
    </div>
  </div>
</form>
`;
